Lib/ntpath.py

```diff
@@ -75,6 +75,10 @@ def isabs(s):
 
 # Join two (or more) paths.
 def join(path, *paths):
+    """Join two or more pathname components, inserting '\\' as needed.
+    If any component is an absolute path, all previous path components
+    will be discarded.  An empty last part will result in a path that
+    ends with a separator."""
```

Comment:
**Formatting**:

Docstrings in the standard library (particularly more recently added ones) should comply with the formatting standards of [PEP 257](https://www.python.org/dev/peps/pep-0257/). In order to make this one compliant, it should contain one line of whitespace in between the first summary sentence and the sentences after it. Also, the final set of triple quotes `"""` should be on a line by itself (when the docstring is more than one line). See the PEP for more details.

**Wording/Details**:

1) "Join two or more pathname components" is not quite correct. Although it's usually used for this purpose, it can also be used on a single path; meaning "one or more" would be more correct in this case.

2) The separator character is inserted in between every path that is not empty; IMO this should be specified instead of just stating "inserting '\\\\' as needed", which is a bit ambiguous. 

3) There is a Windows-specific paragraph of the documentation for `os.path.join()`, which I think should be included (at least partially) in the docstring here, since this section is specifically for the Windows implementation. For reference, here's the paragraph:

```
On Windows, the drive letter is not reset when an absolute path component (e.g., r'\foo') is encountered. If a
component contains a drive letter, all previous components are thrown away and the drive letter is reset.
Note that since there is a current directory for each drive, os.path.join("c:", "foo") represents a path relative to
the current directory on drive C: (c:foo), not c:\foo.
```

For conciseness, I think the last sentence can be excluded, but the rest should probably be in the docstring. I'll admit that part is a bit subjective though.